azerothcore/azerothcore-wotlk
# fix (db): Remove unused script

![image](https://user-images.githubusercontent.com/16887899/195602830-50e0f4d5-a14f-4150-a1aa-552abb60d79f.png)

## Changes Proposed:
-  Removes unused Script Name

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Ran on Database World
- Observered no Red


## How to Test the Changes:


1. run on database
2. observe no red


## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
